ogusa/parameters.py

```diff
@@ -366,44 +370,39 @@ def get_parameters(test=False, baseline=False, guid='',
         err = "Gov't spending rule dates are inconsistent"
         raise RuntimeError(err)
 
-
-
-    # Tax parameters:
-    #   Income Tax Parameters
-    #  will call tax function estimation function here...
-    # do output such that each parameters is in a separate SxBW array
-    # read in estimated parameters
-    #print 'baseline is:', baseline
+    # Income Tax Parameters
+    tx_func_est_path = os.path.join(
+        output_base, 'TxFuncEst_{}.pkl'.format(guid),
+    )
+    if run_micro:
+        txfunc.get_tax_func_estimate(BW, S, starting_age, ending_age,
+                                     baseline=baseline,
+                                     analytical_mtrs=analytical_mtrs,
+                                     age_specific=age_specific,
+                                     start_year=start_year,
+                                     reform=reform, guid=guid,
+                                     tx_func_est_path=tx_func_est_path,
+                                     data=data)
     if baseline:
-        if tx_func_est_path is None:
-            baseline_pckl = "TxFuncEst_baseline{}.pkl".format(guid)
-            estimate_file = os.path.join(TAX_ESTIMATE_PATH,
-                                         baseline_pckl)
-        else:
-            baseline_pckl = "TxFuncEst_baseline{}.pkl".format(guid)
-            estimate_file = tx_func_est_path
+        baseline_pckl = "TxFuncEst_baseline{}.pkl".format(guid)
+        estimate_file = tx_func_est_path
         print 'using baseline tax parameters'
         dict_params = read_tax_func_estimate(estimate_file, baseline_pckl)
 
     else:
-        if tx_func_est_path is None:
-            policy_pckl = "TxFuncEst_policy{}.pkl".format(guid)
-            estimate_file = os.path.join(TAX_ESTIMATE_PATH,
-                                         policy_pckl)
-        else:
-            policy_pckl = "TxFuncEst_policy{}.pkl".format(guid)
-            estimate_file = tx_func_est_path
+        policy_pckl = "TxFuncEst_policy{}.pkl".format(guid)
+        estimate_file = tx_func_est_path
         print 'using policy tax parameters'
```

Comment:
@jdebacker asked about `test_tx_func_est_path`
> @hdoupe I need some help fixing this test. I'm a bit confused by what it is trying to do as it's not testing directly a function in `parameters.py`. Do I need some pickle files sitting in specific locations?

I created that test to check the if statement logic that I added here. I wanted to check that the correct file paths were passed to `read_tax_func_estimate`. However, it looks like you are limiting the tax function parameter file paths to the form `"TxFuncEst_policy{}.pkl"`. So, I don't think the test is needed anymore.

Reply:
Thanks for the suggestion @hdoupe.  I've removed the test.